### PR TITLE
Inomurko/gnosis multisig

### PR DESCRIPTION
**Pull request by InoMurko**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -75,6 +75,37 @@ jobs:
...
+          command: make init_multisig
+          working_directory: ~/repo/MultiSigWalletOverride
+      - run:
+          name: Deploy Gnosis MultiSigWalet
+          command: |
+            sudo apt-get update && sudo apt-get install -y libudev-dev libusb-1.0-0 libusb-1.0-0-dev
+            npm install -g npm@latest
+            nvm use node
+            npm install
+            npx truffle migrate 0xa508dD875f10C33C52a8abb20E16fc68E981F186 1
```
Comment (boolafish) [inline .circleci/config.yml:107]: btw, where does this address comes from? `0xa508dD875f10C33C52a8abb20E16fc68E981F186`

File: README.md [hunk 2]
```diff
@@ -90,6 +90,28 @@ docker run --rm omisego/plasma-contracts npx truffle migrate --network remote
 
 For more detail of the deploying scripts and the env vars to set, see [deploying.md](./plasma_framework/docs/deploying/deploying.md)
 
+The `MAINTAINER` account can also be a multisignature contract. The example we've built uses Gnosis MultiSigWallet. The project has been added as a git submodule:
```
Comment (boolafish) [inline README.md:93]: can we give a link to their repo too in this section? https://github.com/gnosis/MultiSigWallet/tree/v1.6.0

File: MultiSigWalletOverride/truffle.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+module.exports = {
```
Comment (boolafish) [inline MultiSigWalletOverride/truffle.js:1]: Do we need to deploy those Multisig contracts to other networks too?? Or we reuse their existing ones (if there are such ones)?

File: .gitignore [hunk 4]
```diff
@@ -48,3 +48,7 @@ coverage.json
 
 #tenderly config 
 tenderly.yaml
+
+MultiSigWallet
+MultiSigWalletOverride/ganache_data/
+plasma_framework/vault_authority
```
Comment (boolafish) [inline .gitignore:54]: might be redundant if we with this PR: https://github.com/omgnetwork/plasma-contracts/pull/679?

File: .circleci/config.yml (hunk 1 above)
Comment (InoMurko) [reply .circleci/config.yml:107]: it's the first address in ganache

File: MultiSigWalletOverride/truffle.js (hunk 3 above)
Comment (InoMurko) [reply MultiSigWalletOverride/truffle.js:1]: aha, yeah, we can add as many networks as we want!

File: .circleci/config.yml (hunk 1 above)
Comment (souradeep-das) [reply .circleci/config.yml:107]: Ohh the deterministic ganache run from docker-compose

File: .circleci/config.yml [hunk 5]
```diff
@@ -75,6 +75,37 @@ jobs:
...
+      - run: |
+            echo 'export NVM_DIR="/opt/circleci/.nvm"' >> $BASH_ENV
+            echo ' [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"' >> $BASH_ENV
+      - run:
+          name: Install node 9
+          command: |
+            curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
+            nvm install v9
+            node --version
+            node -v
```
Comment (souradeep-das) [inline .circleci/config.yml:87]: extra?

**Pull request merged**